### ISSUE TITLE
Add Vault submodule

### ISSUE THREAD

**Issue by rgarrigue** [terraform-aws-modules/terraform-aws-security-group#192]
Hi there

You already have a submodule for Hashicorp Consul & Nomad, mind adding Vault ? It's listening on port 8200 & 8201 TCP.

Best regards,

**Comment by antonbabenko** [comment]
Good idea, a pull request is welcome. It is very easy to add a new rules group - https://github.com/terraform-aws-modules/terraform-aws-security-group#how-to-addupdate-rulesgroups.

I assume one should be open to more destinations than another, if I understand this correctly - https://learn.hashicorp.com/tutorials/vault/reference-architecture#network-connectivity-details

**Comment by rgarrigue** [comment]
@antonbabenko PR done. One remark though, adding a quick doc about how to contribute (install json2hcl, install terraform-docs, install pre-commit, run it on all files iirc) would help contributors.
